src/providers/WalletProvider/UTXOProvider.tsx

```diff
@@ -0,0 +1,24 @@
+'use client';
+import {
+  BigmiProvider,
+  createDefaultBigmiConfig,
+  useReconnect,
+} from '@lifi/wallet-management';
+import { type FC, type PropsWithChildren } from 'react';
+
+const { config, connectors } = createDefaultBigmiConfig({
```

Comment:
`'connectors' is declared but its value is never read.ts(6133)
`
```suggestion
const { config } = createDefaultBigmiConfig({
```